### PR TITLE
Remove Rounding of Scaled Tier Amount

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/4467

Peek 2021-07-19 09-05

### CODE REVIEW

**Review by znarf — APPROVED** [review]
I think we should get rid of the `K` notation altogether but it's a good step forward.

**Comment by SudharakaP** [comment]
> I think we should get rid of the `K` notation altogether but it's a good step forward.

Hmm... You mean only remove K notation but have other scaling notations such M and stuff. So 1000 will show up as 1000 but 1000000 will show up as 1M? 🤔

**Comment by znarf** [comment]
@SudharakaP No, let me rephrase: I think we should stop trying to abbreviate numbers, at least in the context of the contribution flow.

**Comment by SudharakaP** [comment]
> @SudharakaP No, let me rephrase: I think we should stop trying to abbreviate numbers, at least in the context of the contribution flow.

@znarf : The problem with this I think is currencies such as YEN will always have very high integer values. So without abbreviating they would look like, 10000 YEN, 200000 YEN and so on. But I'll open an issue to discuss. 🤔

**Comment by SudharakaP** [comment]
Done at, https://github.com/opencollective/opencollective/issues/4471

**Comment by SudharakaP** [comment]
I noticed that there can be a certain rounding bug introduced by this PR for certain amounts; for example, 

image

However that's been taken car of at, https://github.com/opencollective/opencollective-frontend/pull/6638 so will not open a separate PR.
